webpack/webpack-env-tests.ts

```diff
@@ -13,3 +13,30 @@ let contextModule = context<SomeModule>('./someModule');
 require(['./someModule', './otherModule'], (someModule: SomeModule, otherModule: any) => {
 
 });
+
+// check if HMR is enabled
+if(module.hot) {
+    // accept update of dependency
+    module.hot.accept("./handler.js", function() {
+        //...
+    });
+}
+
+module.exports = null;
+
+// check if HMR is enabled
+if(module.hot) {
+
+    // accept itself
+    module.hot.accept();
+
+    // dispose handler
+    module.hot.dispose(function() {
+        // revoke the side effect
+        //...
+    });
```

Comment:
Could use a few more calls from the other newly added methods. Especially those `data` usages.